fm.cancel.Ripcord.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: fm.cancel.Ripcord
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download
+
+command: Ripcord
+
+modules:
+  - name: Ripcord
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://cancel.fm/dl/Ripcord-0.4.29-x86_64.AppImage
```

Comment:
you build an aarch64 flatpak with a x86_64 binary

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64